lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -310,7 +315,14 @@ def missing_key_helper(keys):
     def test_bad_dates(self, client, server_config, pbench_token):
         """Test behavior when a bad date string is given."""
         if self.api_method != ApiMethod.POST:
-            pytest.skip("skipping " + self.test_bad_dates.__name__)
+            # Only POST APIs have a payload.
```

Comment:
A true statement, but not really descriptive in this context. We're actually assuming here that we can't have a `DATE` param type for a URI or query parameter. We *could*, but currently we *don't*. 😦

Reply:
The code below (line 333 in the changed code, line 321 in the original) looks for a parameter of type `DATE` only in `self.payload`.  So that is why the comment says "payload".